graphql/parameter_queries.graphql

```diff
@@ -73,3 +73,15 @@ mutation UpsertParameterMutation($projectId: ID, $environmentName: String, $keyN
     }
   }
 }
+
+mutation DeleteParameterMutation($parameterId: ID!) {
```

Comment:
The query files have all been sorted by the query name. If you'd rather just append, that's fine too.